### PR TITLE
Configurable expanded labels on cards

### PR DESCRIPTION
**Pull request by rauhryan**

### CODE REVIEW

File: ember-app/app/initializers/punch-component.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import Ember from 'ember';
```
Comment (discorick) [inline ember-app/app/initializers/punch-component.js]: Not familiar with the "punch" verbiage, curious where it comes from?

**Comment by discorick** [comment]
I like the setting for expanded labels being global, it helps keep the visualization of cards consistent across boards. The only potential downside is how it expands milestones as well... I wonder if users will still want an expanded view for the taskboard, but more compact on milestones or vice versa.

**Comment by discorick** [comment]
image

Minor, PR icon is missing purple styling.

**Comment by rauhryan** [comment]
> I like the setting for expanded labels being global, it helps keep the visualization of cards consistent across boards. The only potential downside is how it expands milestones as well... I wonder if users will still want an expanded view for the taskboard, but more compact on milestones or vice versa.

We can ship and see if anyone gives that feedback

**Comment by rauhryan** [comment]
I've addressed all feedback, so I'm going to merge

**Pull request merged**